app/logic/searchVolunteersAndStudents.py

```diff
@@ -3,8 +3,7 @@
 from flask import json, jsonify
 
 
-
-def searchVolunteers(query):
+def searchVolunteersAndStudents(query):
```

Comment:
searchVolunteersAndStudents() is too wordy. It should reflect what you are actually doing in the function, not how it will be used. So, `searchStudents()`, or even `searchUsers()`.

Reply:
Also, don't mess with JSON here. Let the controller handle changing the representation if it needs to. It's easier to use and easier to test if the function returns a dictionary.

Reply:
changed `searchVolunteersAndStudents() ` to `searchUsers()`. Moved JSON to controller and changed test accordingly.